pandas/core/series.py

```diff
@@ -4447,6 +4447,35 @@ def fillna(
             downcast=downcast,
         )
 
+    @Appender(
+        """
+        Parameters
+        ----------
+        item : int
```

Comment:
Does `item` have to be an `int`? For example, the index could be made of strings...I think `label` would be correct

Reply:
Yes sure. I haven't thought of these edge case.